com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumeVoxelization.compute

```diff
@@ -136,6 +136,9 @@ void FillVolumetricDensityBuffer(PositionInputs posInput, uint tileIndex, Jitter
                 // We have found the first density volume.
                 break;
             }
+
+            if( offset >= UINT_MAX )
```

Comment:
will suggest to do 
#if defined(SHADER_API_METAL)
 if( offset >= UINT_MAX ) { break; } ; 
#endif

(if this workaround work :) )